fix failing CI

k-yle opened the pull request:
Closes https://github.com/openstreetmap/id-tagging-schema/issues/1848

A new langauge was added to transifex recently: [`be@tarask`](https://en.wikipedia.org/wiki/Tara%C5%A1kievica). Due to the `@` symbol, this code isn't a valid [BCP 47 language tag](https://developer.mozilla.org/en-US/docs/Glossary/BCP_47_language_tag), so [`Intl.Locale`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale) is throwing an error.

k-yle committed:
fix failing CI

matkoniecz commented:
note that some codes are rematched in https://github.com/openstreetmap/iD/blob/develop/scripts/language_names.js#L5

no idea whether putting tarask there would suffice

k-yle commented:
> no idea whether putting tarask there would suffice

yeah, that would also solve the problem, but then the next time a new [language with an `@` symbol](https://explore.transifex.com/languages#:~:text=@) is added, it'll break again.

So i feel like making the code more robust would be a better solution

tyrasd committed:
fix accessing of transifex strings

In this variable, the strings are still indexed by their "raw" transifex language code. Without this, the output for normalized languages would be just the empty fallback (`{}`).

tyrasd reviewed (approved):
Found (and fixed) a small bug: at line 125 we need to use the "raw" language code, because `allStrings` still is indexed by whatever transifex returned). Otherwise the content of the `be-tarask` output is always empty.

In general: I found this language a bit hard to test, as none of my usual browsers let me set `be-tarask` as my preferred language.[^1] But it did work as expected in the openstreetmap-website integration when selecting `be-tarask` from the language selector there.

[^1]: in Firefox, I was able to manually override the `intl.accept_languages` setting via `about:config`
